src/elements/header/header/header.stories.ts

```diff
@@ -50,6 +61,9 @@ const HeaderBasicTemplate = (
       <sbb-menu-button>Italiano</sbb-menu-button>
       <sbb-menu-button icon-name="tick-small">English</sbb-menu-button>
     </sbb-menu>
+    ${args.size === 's'
+      ? html`<sbb-signet slot="logo" protective-room="none" padded></sbb-signet>`
```

Comment:
probably we should add also the link (<a>) to the example as in most cases it is the link to return to home. I feel that in readme we should provide this example as well to show how to use the signet instead of the logo

Reply:
@kyubisation, should we update all our examples/tests without the logo slot starting now (and provide a utility class for the gap between the logo and the remaining content)? Or is it better with the in-between state (standard logo still displayed as standard in slot, but for size s you have to slot the new signet into the logo slot, like Davide implemented)?
I have the feeling to change it now, as then we never provided any documentation with slotting the signet into the logo slot.